### PR TITLE
Detect keyword 'linear' on lines containing word 'animation' too

### LINKED ISSUE #10984: CSS Timing Function Quick Edit: linear is invalid syntax
Steps to reproduce: 

* Create a .css file

``` css
.test {
  animation-timing-function: linear;
}
```

* Place cursor on linear and press CTRL+E, which results in `CSS Timing Function Quick Edit: invalid syntax`. 

Expected: Timing Function Quick Edit (Bezier curve editor?) view opens up with a line representing `cubic-bezier(0.0, 0.0, 1.0, 1.0)` [(ref MDN)](https://developer.mozilla.org/en/docs/Web/CSS/timing-function#linear).

Windows 8.1, Release 1.4 development build 1.4.0-0 (master 9264e29c0)

### PR DESCRIPTION
**Pull request by petetnt**
Fixes #10984

### CODE REVIEW

File: src/extensions/default/InlineTimingFunctionEditor/TimingFunctionUtils.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ define(function (require, exports, module) {
         BEZIER_CURVE_GENERAL_REGEX      = /cubic-bezier\((.*)\)/,
         EASE_STRICT_REGEX               = /[: ,]ease(?:-in)?(?:-out)?[ ,;]/,
         EASE_LAX_REGEX                  = /ease(?:-in)?(?:-out)?/,
-        LINEAR_STRICT_REGEX             = /transition.*?[: ,]linear[ ,;]/,
+        LINEAR_STRICT_REGEX             = /transition|animation.*?[: ,]linear[ ,;]/,
```
Comment (MarcelGerber) [inline src/extensions/default/InlineTimingFunctionEditor/TimingFunctionUtils.js]: This should actually be between parenthesis, like this: `(transition|animation).*?[: ,]linear[ ,;]`

**Comment by MarcelGerber** [comment]
Could you also add a [unit test](https://github.com/adobe/brackets/blob/master/src/extensions/default/InlineTimingFunctionEditor/unittests.js) for this?

File: src/extensions/default/InlineTimingFunctionEditor/TimingFunctionUtils.js (hunk 1 above)
Comment (petetnt) [inline src/extensions/default/InlineTimingFunctionEditor/TimingFunctionUtils.js]: Yeah you are correct there, don't remember what I was thinking there!

**Comment by petetnt** [comment]
@MarcelGerber for the unit tests, [should these tests](https://github.com/adobe/brackets/blob/master/src/extensions/default/InlineTimingFunctionEditor/unittests.js#L137) be just repeated with the `animation` keyword or should the existing tests be modified to test for both the ´transition"  and "animation" keywords?

**Comment by MarcelGerber** [comment]
It's probably enough to just repeat one or two unit tests with the `animation` keyword.

**Comment by petetnt** [comment]
Added the unit tests

**Comment by MarcelGerber** [comment]
Thanks!
Merging.

**Comment by petetnt** [comment]
:+1: awesome

**Pull request merged**